### DISCUSSION TITLE
IEEE 802.11 Block Acknowledgment - Showcase

### DISCUSSION THREAD

**Discussion by adamgeorge309** [inet-framework/inet#620] [Showcases]
Discuss the "IEEE 802.11 Block Acknowledgment" showcase here. Ideas and recommendations are welcome.

**Comment by Jaiabhisheksharmaa** [comment]
Hi , I have a query. 
the 802.11ax std states that the 'BA bitmap length for the compressed BA frame' should be less than the 'negotiated buffer size'. However, in the table 26-1 row no. 2 shows that 'negotiated buffer size is 64 or 128' for 'BA length  64 or 256' .  here the BA length can be 256 which is greater than the 128bits. can you please explain this.
